monix-catnap/src/main/scala/com/avast/sst/monix/catnap/CircuitBreakerModule.scala

```diff
@@ -0,0 +1,32 @@
+package com.avast.sst.monix.catnap
+
+import cats.effect.{Clock, Sync}
+import monix.catnap.CircuitBreaker
+
+class CircuitBreakerModule[F[_]](implicit F: Sync[F]) {
```

Comment:
would this trick also work if the class were private?

Reply:
I don't think so, you definitely need some "interface" where you can access the `make` method.

Reply:
just for curiosity's sake, could you try it? I have small suspicion that it might work, because for that trick, you don't need to create a `CircuitBreakerModule` instance from outside

Reply:
No, this doesn't work. I tried but I knew it couldn't be done. You get a compilation error `Error:(53, 26) private class CircuitBreakerModule escapes its defining scope as part of type com.avast.sst.monix.catnap.CircuitBreakerModule[F]
  def apply[F[_]: Sync]: CircuitBreakerModule[F] = new CircuitBreakerModule[F]`